README.md

```diff
@@ -139,6 +142,13 @@ To efficiently parse messages, Homie defines a few rules related to topic names.
     <td>Yes or No, depending of your implementation</td>
     <td>No</td>
   </tr>
+  <tr>
+    <td>$nodes</td>
+    <td>Device → Controller</td>
+    <td>Nodes the device exposes, with format `id` separated by a `,` if there are multiple nodes. For ranges, define the range after the `id`, within `[]` and separated by a `-`.</td>
```

Comment:
Can you replace the `` ` `` with `<code>` tags? The `` ` `` doesn't work in HTML arrays.